app/src/main/java/ch/hikemate/app/ui/components/HikeCard.kt

```diff
@@ -99,11 +100,15 @@ fun HikeCard(
                     elevationData = elevationData,
                     modifier =
                         Modifier.weight(1f)
+                            .zIndex(1.0f)
                             .fillMaxHeight()
                             .background(
                                 MaterialTheme.colorScheme.primary.copy(alpha = 0.1f),
                                 RoundedCornerShape(4.dp)),
```

Comment:
Suggestion: I tried removing the background and IMO it looks way better now that you filled the graphs. Could you remove it ?

Reply:
It isn't in the scope of this task to change the figma. As you stated in a previous comment, the code should reflect the figma, so we might want to change the figma first and then create a new task for it!

Reply:
Yep, you're right!

Reply:
I reduced the stroke to make it less aggressive then